typing/btype.ml

```diff
@@ -260,7 +260,7 @@ let fold_row f init row =
       (row_fields row)
   in
   match get_desc (row_more row) with
-  | Tvar _ | Tunivar _ | Tsubst _ | Tconstr _ | Tnil ->
+  | Tvar _ | Tunivar _ | Tsubst _ | Tconstr _ | Tnil | Tof_kind _ ->
```

Comment:
This scares me a little. I don't have a semantics for when `Tof_kind` appears in a `row_more` field. Do we need this? If so, I'd like to understand the case better.